#! /bin/sh /usr/share/dpatch/dpatch-run
## collection_conf_path.dpatch by Sebastian Harl <sh@tokkee.org>
##
## DP: Set the path of collection.conf to /etc/collectd/.

@DPATCH@

diff a/contrib/collection.cgi b/contrib/collection.cgi
--- a/contrib/collection.cgi
+++ b/contrib/collection.cgi
@@ -11,7 +11,7 @@ use URI::Escape ('uri_escape');
 use RRDs ();
 use Data::Dumper ();
 
-our $Config = "/etc/collection.conf";
+our $Config = "/etc/collectd/collection.conf";
 our @DataDirs = ();
 our $LibDir;

